### ISSUE TITLE
[Combobox] Combobox does not respect property setInitialFocus: true in callout props  and does not provide the way to highlight the first selection in dropdown  for combobx with no autocomplete as it should be via Aria guidlines

### ISSUE THREAD

**Issue by chtoosha** [microsoft/fluentui#13958]
### Environment Information

- **Package version(s)**: 7.110.5
- **Browser and OS versions**: WIN 10 + Chrome, Win 10 + Electron

### Describe the issue:

According to  https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html 
"When visual focus is in the listbox, DOM focus remains on the combobox and the value of aria-activedescendant on the combobox is set to a value that refers to the listbox option that is visually indicated as focused. Where the following descriptions of keyboard commands mention focus, they are referring to the visual focus indicator. For more information about this focus management technique, see Using aria-activedescendant to Manage Focus."

However that does not happen if user deals with slect only combobox. 

I have tried to use 
calloutProps={{
          setInitialFocus: true,
                     }}
      />
 but it seems like that setting is not respected

#### Actual behavior:
No visual focus on first element 

#### Expected behavior:
Visual focus should be settled to fist element.

### If applicable, please provide a codepen repro:

https://codepen.io/aleksandra-potapova/pen/pogVrJr

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) 
No

Requested priority: (Blocking, High, Normal, Low)
Normal

Products/sites affected: (if applicable)

**Comment by paulgildea** [comment]
@chtoosha - I took a look at the ARIA spec (while I'm not an expert) - I think the desired behavior you are looking for is in our `Dropdown` component - https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown

The `ComboBox` inherently allows the user to type in arbitrary text and the Dropdown fits more the select only model.

Let me know if that helps.

**Comment by chtoosha** [comment]
@paulgildea Thank you for the rapid answer. 
I have a couple of questions:
- What is "select only model"?
- Isn't it a bug that a bug thta  _calloutProps={{
setInitialFocus: true,
}}_ 
Is not respected?

**Comment by paulgildea** [comment]
For your #1 - I was referring to the ARIA example you provided: https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html that example calls it select only - meaning that you cannot type any text into the ComboBox.

Today we have two different components to manage that. `ComboBox` and `Dropdown`. The `Dropdown` component does not allow text to be typed in and fits the select only model.

For your #2 - I'm unclear - @joschect do you have any clarity on the API? @xu as FYI.

**Comment by chtoosha** [comment]
Any update?
